packages/ember-data/lib/system/relationships/belongs-to.js

```diff
@@ -89,19 +89,21 @@ function belongsTo(modelName, options) {
 
   return computedPolyfill({
     get: function(key) {
-      return this._relationships[key].getRecord();
+      return this._internalModel._relationships[key].getRecord();
```

Comment:
As soon as I merge this, the lazy relationships refactor is next, there is a pr for it, it actually makes sense now that models are lazy, so that will make this be nicer